src/06-loops.js

```diff
@@ -25,7 +33,17 @@ function rangeSum(min, max) {}
  *  rangeOdd(10, 15);
  *  //> [ 15, 13, 11 ]
  */
-function rangeOdd(min, max) {}
+function rangeOdd(min, max) {
+  arr = [];
```

Comment:
oh noes!!! we forgot to declare our variable here and accidentally created a global variable!